### ISSUE TITLE
RelationFieldNotFound on nested create/connect

### ISSUE THREAD

**Issue by Errorname** [prisma/prisma2#218]
I get a "RelationFieldNotFound" when trying to do a nested create/connect.

The strange thing is that the error says it couldn't found the relation field for "user" on model "Question". But there isn't supposed to be one and I'm not trying to create one.

**schema.prisma**
```prisma
datasource db {
  provider = "sqlite"
  url      = "file:default$default.db"
  default  = true
}

generator photon {
  provider = "photonjs"
}

model Question {
  id     String  @default(cuid()) @id @unique
  title  String
  answer Answer?
}

model Answer {
  id       String    @default(cuid()) @id @unique
  content  String
  question Question?
  user     User
}

model User {
  id      String   @default(cuid()) @id @unique
  name    String?
  answers Answer[]
}
```

**index.js**
```js
const user = await photon.users.create({
    data: {
      name: 'The User'
    }
  })

  const node = await photon.questions.create({
    data: {
      title: 'The Question',
      answer: {
        create: {
          content: 'The Answer',
          user: {
            connect: {
              id: user.id
            }
          }
        }
      }
    }
  })

  console.log({ node })
```

**Console**
```
DomainError(RelationFieldNotFound { name: "user", model: "Question" })
    at PhotonFetcher.<anonymous> (/Users/errorname/Documents/Vrac/hello-prisma2/nested-create/node_modules/@generated/photon/index.js:34:23)
    at Generator.throw (<anonymous>)
    at rejected (/Users/errorname/Documents/Vrac/hello-prisma2/nested-create/node_modules/@generated/photon/index.js:5:65)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by Errorname** [comment]
It looks like a duplicate of https://github.com/prisma/photonjs/issues/33

**Comment by timsuchanek** [comment]
@divyenduz please check if the issues are indeed related.

**Comment by divyenduz** [comment]
I am able to reproduce this: https://github.com/divyenduz/p2_218

The following error is observed: 

```
(node:49200) UnhandledPromiseRejectionWarning: Error: Error in Photon: 
DomainError(RelationFieldNotFound { name: "user", model: "Question" })
    at PhotonFetcher.<anonymous> (/Users/divyendusingh/Documents/prisma/p2-218/node_modules/@generated/photon/index.js:35:23)
    at Generator.throw (<anonymous>)
    at rejected (/Users/divyendusingh/Documents/prisma/p2-218/node_modules/@generated/photon/index.js:5:65)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:49200) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:49200) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**Comment by divyenduz** [comment]
I don't think that these are related issues, but @pantharshit00 would post the error message from the issue issue shortly and we can confirm the hypothesis.

**Comment by mavilein** [comment]
This is fixed in binary version `b8957df02bca6708b7aefa56832eda409dbb1114` which is available on `alpha`.
